### PR TITLE
feat(InputNumber): adding label element in InputNumber Component and …

### PR DESCRIPTION
**Pull request by Sofien-Sellami**
…export it

### Context
Export the InputNumber component and adding a l label 

### Changes & Results
Capture d’écran du 2023-06-29 17-02-29

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by wayfarer3130** [comment]
Could we please have some images showing what this looks like?

**Comment by salimkanoun** [comment]
@wayfarer3130 here it is (but deploy preview has been cancled ? )

**Comment by salimkanoun** [comment]
here it is
image

File: platform/ui/src/components/InputNumber/InputNumber.tsx [hunk 1]
```diff
@@ -14,6 +15,7 @@ import './InputNumber.css';
 
 const sizesClasses = {
   sm: 'w-[45px] h-[28px]',
+  bg: 'w-[206px] h-[35px]',
```
Comment (sedghi) [inline platform/ui/src/components/InputNumber/InputNumber.tsx:18]: if by bg you mean big? I prefer lg to be more consistent with the rest of the code

**Comment by sedghi** [comment]
can you fix the CI error?

File: platform/ui/src/components/InputNumber/InputNumber.tsx [hunk 2]
```diff
@@ -76,6 +82,8 @@ const InputNumber: React.FC<{
   );
 
   return (
+  <div className={'flex flex-col flex-1'}>
+          <Label className={labelClassName} text={label}></Label>
```
Comment (sedghi) [inline platform/ui/src/components/InputNumber/InputNumber.tsx:86]: this should have a gaurd for the label 

```js
label &&
```

**Pull request merged**